joomla/joomla-cms
# [Hathor] Send test mail button missing from config

#### Summary of Changes

Added the new send test mail button to the global config in the hathor template
#### Testing Instructions

Apply patch, switch to the hathor template and check that the send test mail button is present and working in the global configuration
